modules/fluentd_pre/main.tf

```diff
@@ -0,0 +1,27 @@
+provider "template" {
+  # See: https://github.com/terraform-providers/terraform-provider-template/blob/v2.0.0/CHANGELOG.md#200-january-14-2019
+  # Need to pin the minimum version for templates/fluent.conf
+  version = "~> 2.0"
+}
+
+data "template_file" "fluentd_conf" {
+  template = file("${path.module}/templates/fluent.conf")
+
+  vars = {
+    elasticsearch_host = var.elasticsearch_host
+    elasticsearch_port = var.elasticsearch_port
+
+    fluentd_port  = 4224
```

Comment:
Since `fluentd_new` uses a var for setting the port value `4224`, consider doing the same for `fluentd_pre` too?

Reply:
Done